### ISSUE TITLE
[1.21.3] forceTicks in addRegionTicket isn't working correctly

### ISSUE THREAD

**Issue by thexaero** [MinecraftForge/MinecraftForge#10184]
**Minecraft Version:** 1.21.3
**Forge Version:** tested on 53.0.19

**Steps to Reproduce:**
 1. Add a ticket with `ServerChunkCache.addRegionTicket(TicketType<T> p_140841_, ChunkPos p_140842_, int p_140843_, T p_140844_, boolean forceTicks)` with forceTicks set to true.
 2. Plant some wheat seeds in the forceloaded chunk.
 3. Fly far away and set random tick speed to 3000, then wait for 30 seconds.
 4. Set random tick speed back to 3.
 5. Return to the forceloaded chunk.
 6. The wheat has not grown.

**Description of issue:**
Forceload tickets with forceTicks set to true don't force random block ticks. I feel like `DistanceManager.shouldForceTicks` isn't even called anywhere for this purpose, but could be wrong here.

**Comment by Jonathing** [comment]
Yup, it looks like chunk ticking was moved to a completely different place, so the original patch that was used to tick forced chunks was eaten. Working on it now.

**Comment by Jonathing** [comment]
The fix for this is good to go. Just for future reference, `DistanceManager.shouldForceTicks` works, but is only used to reflect which tickets are part of the `forcedTickets` variable. The forced chunks themselves are extracted from the tickets in `ServerChunkCache` using `ChunkMap.forEachForcedChunk`. You can see the code in the relevant PRs.
